mantidproject/mantid
# 21176 mechamism to check is specified workspace exists in sans2 d gui

When a no-existent workspace was entered the sans GUI produces the full stack trace from the line where it could not find the file. This catches the exception and produces a more concise informative error message instead. 

**To test:**
- Open the sans GUI V2 and load any user file. For example the user file from the loqdemo folder which is part of the ISIS sample data which can be downloaded from http://download.mantidproject.org/. It is called maskfile.
- Then enter '57487583' in the first column of the first row of the table making sure to deselect the cell afterwards.
- Click process you should see the following message _Process halted due to: There was a bad entry for row 0. Ensure that the path to your files has been added to the Mantid search directories! See here for more details: Trying to find the SANS file 89809890, but cannot find it. Make sure that the relevant paths are added and the correct instrument is selected._


Fixes #21176 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
